cfgov/v1/views/reports.py

```diff
@@ -41,7 +42,8 @@ class PageMetadataReportView(PageReportView):
         ("language", "Language"),
         ("search_description", "Search description"),
         ("tags.names", "Tags"),
-        ("categories.all", "Categories")],
+        ("categories.all", "Categories"),
+        ("content_owners.names", "Content Owner(s)")],
```

Comment:
I think for database query optimization purposes (and consistency) it would be good to add `"content_owners"` to the `prefetch_related()` call below on line 64. This tells Django to do a single query to prefetch the list of content owners for all of the pages, as opposed to having to do one query per page. [Some background reading](https://scoutapm.com/blog/django-and-the-n1-queries-problem) if you're interested.